app/assets/javascripts/table-sort.es6

```diff
@@ -0,0 +1,78 @@
+$(document).ready(function() {
+  if ($('table.sortable')) {
+    let table;
+    $('th.sort')
+      .wrapInner('<span class="sortable-column" title="sort this column"/>')
+      .each(function() {
+        const th = $(this),
+          thIndex = th.index();
```

Comment:
I think the[ jQuery function `each`](http://api.jquery.com/jquery.each/) passes the index and current element to the inner function, so I don't think you should have to grab them again.:
```javascript
.each(function(thIndex, th) {
```

Reply:
good point, I'll try that